src/AbstractFixture.php

```diff
@@ -17,9 +19,9 @@ abstract class AbstractFixture implements SharedFixtureInterface
     /**
      * Fixture reference repository
      *
-     * @var ReferenceRepository
+     * @var ReferenceRepository|null
      */
-    protected $referenceRepository;
+    protected $referenceRepository = null;
```

Comment:
Untyped properties are `null` by default and don't need a `null` initialization. The docblock change should be enough.